packages/cloud_firestore_odm/cloud_firestore_odm/example/integration_test/common.dart

```diff
@@ -44,5 +45,21 @@ Movie createMovie({
   );
 }
 
+Enums createEnums({
```

Comment:
I'm not sure what you mean. I just emulated the `createMovie` function above for the tests

Reply:
createMovie exists because some tests want the fields to be required (such as to test if they are missing)

We don't care about this for enum tests.